src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/CompleteConfigurationTest.php

```diff
@@ -111,7 +111,6 @@ public function testFirewalls()
                     'remote_user',
                     'form_login',
                     'http_basic',
-                    'http_digest',
```

Comment:
Instead of removing this now, we could mark the corresponding test with `@group legacy`, but the test case must not be removed in 4.0.

Reply:
Or copy fixture `container1.*` as `container1_with_digest.*` and duplicate the test so that it can be removed in 4.0? (at least for this one)

Reply:
Ok, done.